app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -149,32 +163,34 @@ public void onClick(View view) {
         });
 
         // Click listeners for changing activity content
-        joystickContentBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (headsetIsConnected && carIsConnected) {
-                    joystickContent.setVisibility(View.VISIBLE);
-                    eegContent.setVisibility(View.GONE);
-
-                    joystickContentBtn.setVisibility(View.GONE);
-                    eegContentBtn.setVisibility(View.VISIBLE);
-
-                    if (eegActive) {
-                        eegActive = false;
-                        stopEeg();
-                        stopGyro();
-                    }
-                }
-                else {
-                    // do nothing
-                }
-            }
-        });
 
-        eegContentBtn.setOnClickListener(new View.OnClickListener() {
+      joystickContentBtn.setOnClickListener(new View.OnClickListener() {
+          @Override
+          public void onClick(View v) {
+           //   if (headsetIsConnected && carIsConnected) {
+//                    //joystickContent.setVisibility(View.VISIBLE);
+//                    eegContent.setVisibility(View.GONE);
+//
+//                    joystickContentBtn.setVisibility(View.GONE);
+//                    eegContentBtn.setVisibility(View.VISIBLE);
+//
+//                    if (eegActive) {
+//                        eegActive = false;
+//                        stopEeg();
+//                        stopGyro();
+//                    }
+//                }
+//                else {
+//                    // do nothing
+//                }
+                  SecondActivity.start(MainActivity.this);
+          }
+      });
+
+            eegContentBtn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                joystickContent.setVisibility(View.GONE);
```

Comment:
This line needs to be included in the onClick listener, because if it isn't there will be too many buttons in the header visible at the same time.